recipes/pyepr/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pyepr" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 72a2044ffcd570a17441d72d625f743b7c962b571b505eab831dcfc1b216f82a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.7
```

Comment:
If you are compiling & linking against numpy, please use the instructions here: https://conda-forge.org/docs/maintainer/knowledge_base.html?#building-against-numpy